webui/react/src/routes/index.ts

```diff
@@ -60,6 +72,7 @@ export const sidebarRoutes: RouteConfigItem[] = [
     path: '/ui/experiments',
     title: 'Experiments',
   },
+  // TaskListRoute,
```

Comment:
chore: drop this line

Reply:
hmm was trying to show we'd flip that once we are ready to show the page, but that's simple enough

Reply:
removed it